website/docs/sql_ddl.md

```diff
@@ -40,6 +40,13 @@ CREATE TABLE IF NOT EXISTS hudi_table (
 ) USING hudi;
 ```
 
+:::note NOTE:
+If you're running this tutorial locally and if you are facing `java.io.IOException: Mkdirs failed to create file:/user/hive/warehouse/hudi_table/.hoodie` issue,
```

Comment:
Also can you add the context on why would this happen? I believe you have described this pull request with this context - `has a previous Spark-HMS integration and if they don't have a non-default database setup or if they don't provide LOCATION with DDL`. We should call out that here as well.

Reply:
done